### ISSUE TITLE
[BUG] null =>  FlySpectate NoSuchMethodError ...

### ISSUE THREAD

**Issue by Oruss7** [slipcor/pvparena#90]
Hi, 

Can not leave the arena (can't break, use etc ....)

Caused by: java.lang.NoSuchMethodError: org.bukkit.Bukkit.getOnlinePlayers()Ljava/util/Collection;

pa version : 1.3.0.548
FlySpectate module 1.3.0.515
craftbukkit 1.7.9-R0.1-10

```
org.bukkit.command.CommandException: Unhandled exception executing command 'pa' in plugin pvparena v1.3.0.548
at org.bukkit.command.PluginCommand.execute(PluginCommand.java:46) ~[craftbukkit-1.7.9-R0.2.jar:git-Bukkit-1.7.9-R0.1-10-g8688bd4-b3092jnks]
at org.bukkit.command.SimpleCommandMap.dispatch(SimpleCommandMap.java:180) ~[craftbukkit-1.7.9-R0.2.jar:git-Bukkit-1.7.9-R0.1-10-g8688bd4-b3092jnks]
at org.bukkit.craftbukkit.v1_7_R3.CraftServer.dispatchCommand(CraftServer.java:701) ~[craftbukkit-1.7.9-R0.2.jar:git-Bukkit-1.7.9-R0.1-10-g8688bd4-b3092jnks]
at net.minecraft.server.v1_7_R3.PlayerConnection.handleCommand(PlayerConnection.java:956) [craftbukkit-1.7.9-R0.2.jar:git-Bukkit-1.7.9-R0.1-10-g8688bd4-b3092jnks]
at net.minecraft.server.v1_7_R3.PlayerConnection.a(PlayerConnection.java:817) [craftbukkit-1.7.9-R0.2.jar:git-Bukkit-1.7.9-R0.1-10-g8688bd4-b3092jnks]
at net.minecraft.server.v1_7_R3.PacketPlayInChat.a(PacketPlayInChat.java:28) [craftbukkit-1.7.9-R0.2.jar:git-Bukkit-1.7.9-R0.1-10-g8688bd4-b3092jnks]
at net.minecraft.server.v1_7_R3.PacketPlayInChat.handle(PacketPlayInChat.java:47) [craftbukkit-1.7.9-R0.2.jar:git-Bukkit-1.7.9-R0.1-10-g8688bd4-b3092jnks]
at net.minecraft.server.v1_7_R3.NetworkManager.a(NetworkManager.java:157) [craftbukkit-1.7.9-R0.2.jar:git-Bukkit-1.7.9-R0.1-10-g8688bd4-b3092jnks]
at net.minecraft.server.v1_7_R3.ServerConnection.c(SourceFile:134) [craftbukkit-1.7.9-R0.2.jar:git-Bukkit-1.7.9-R0.1-10-g8688bd4-b3092jnks]
at net.minecraft.server.v1_7_R3.MinecraftServer.v(MinecraftServer.java:667) [craftbukkit-1.7.9-R0.2.jar:git-Bukkit-1.7.9-R0.1-10-g8688bd4-b3092jnks]
at net.minecraft.server.v1_7_R3.DedicatedServer.v(DedicatedServer.java:260) [craftbukkit-1.7.9-R0.2.jar:git-Bukkit-1.7.9-R0.1-10-g8688bd4-b3092jnks]
at net.minecraft.server.v1_7_R3.MinecraftServer.u(MinecraftServer.java:558) [craftbukkit-1.7.9-R0.2.jar:git-Bukkit-1.7.9-R0.1-10-g8688bd4-b3092jnks]
at net.minecraft.server.v1_7_R3.MinecraftServer.run(MinecraftServer.java:469) [craftbukkit-1.7.9-R0.2.jar:git-Bukkit-1.7.9-R0.1-10-g8688bd4-b3092jnks]
at net.minecraft.server.v1_7_R3.ThreadServerApplication.run(SourceFile:628) [craftbukkit-1.7.9-R0.2.jar:git-Bukkit-1.7.9-R0.1-10-g8688bd4-b3092jnks]
Caused by: java.lang.NoSuchMethodError: org.bukkit.Bukkit.getOnlinePlayers()Ljava/util/Collection;
at net.slipcor.pvparena.modules.flyspectate.FlySpectate.unload(FlySpectate.java:153) ~[?:?]
at net.slipcor.pvparena.loadables.ArenaModuleManager.unload(ArenaModuleManager.java:265) ~[?:?]
at net.slipcor.pvparena.arena.PlayerState.unload(PlayerState.java:171) ~[?:?]
at net.slipcor.pvparena.arena.Arena.resetPlayer(Arena.java:1223) ~[?:?]
at net.slipcor.pvparena.arena.Arena.removePlayer(Arena.java:1009) ~[?:?]
at net.slipcor.pvparena.arena.Arena.playerLeave(Arena.java:842) ~[?:?]
at net.slipcor.pvparena.arena.Arena.stop(Arena.java:1514) ~[?:?]
at net.slipcor.pvparena.commands.PAA_Reload.commit(PAA_Reload.java:41) ~[?:?]
at net.slipcor.pvparena.PVPArena.onCommand(PVPArena.java:289) ~[?:?]
at org.bukkit.command.PluginCommand.execute(PluginCommand.java:44) ~[craftbukkit-1.7.9-R0.2.jar:git-Bukkit-1.7.9-R0.1-10-g8688bd4-b3092jnks]
... 13 more

```

**Comment by slipcor** [comment]
soon I will have a jenkins build that fixes this by compiling against 1.7.9 :)

**Comment by slipcor** [comment]
can you please try this?

https://ci2.craftyn.com/job/PVP%20Arena%20Modules/lastSuccessfulBuild/artifact/m_FlySpectate/target/
